include/usgscsm/UsgsAstroLsSensorModel.h

```diff
@@ -916,43 +915,25 @@ class UsgsAstroLsSensorModel : public csm::RasterGM, virtual public csm::Settabl
       double* achievedPrecision = NULL,
       csm::WarningList* warnings = NULL) const;
 
-   // methods pulled out of los2ecf and computeViewingPixel
+   void calculateAttitudeCorrection(
+     const double& time,
+     const std::vector<double>& adj,
+     double attCorr[9]) const;
 
-   void computeDistortedFocalPlaneCoordinates(
-       const double& line,
-       const double& sample,
-       double& distortedLine,
-       double& distortedSample) const;
+   void reconstructSensorDistortion(
+     double& focalX,
+     double& focalY,
+     const double& desiredPrecision) const;
 
    void computeUndistortedFocalPlaneCoordinates(

```

Comment:
Why was this one not moved to the Utilities class? Was it just not used elsewhere?

Reply:
`reconstructSensorDistortion` is removed and replaced with one of the distortion functions that @acpaquette pulled out in PR #172. I just left it as-is in this PR so that tests would pass.